foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/spring/ImportCheckTest.java

```diff
@@ -0,0 +1,65 @@
+package com.sap.ai.sdk.foundationmodels.openai.spring;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.github.javaparser.JavaParser;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+
+class ImportCheckTest {
```

Comment:
(Minor)

I'm not a fan of this such much code for a mundane import check. And now it's even duplicating per module.

See comparison for regex check `^import org\.springframework\.ai\..*` in files with path matching `^(?!.*spring).+$` (i.e. no spring).